modules/sc-mesh-secure-deployment/src/2_0/features/cbma/auth/authServer.py

```diff
@@ -51,14 +55,14 @@ def handle_client(self, client_connection, client_address):
 
     def authenticate_client(self, client_connection, client_address, client_mac):
         secure_client_socket = self.context.wrap_socket(client_connection, server_side=True)
+        secure_client_socket.settimeout(self.CLIENT_TIMEOUT)
```

Comment:
CLIENT* named variable may looks a little misleading here, withing the Server class )

Reply:
Done :+1: